.github/templates/macos_binaries_workflow.yml.j2

```diff
@@ -0,0 +1,68 @@
+{% import 'common.yml.j2' as common %}
```

Comment:
This file is not really related to python 3.10. Sould it be included in this PR ?

Reply:
Good point, let me update